boa/src/builtins/string/tests.rs

```diff
@@ -533,34 +533,46 @@ fn test_match() {
 #[test]
 fn trim() {
     let mut context = Context::new();
-    assert_eq!(forward(&mut context, "'Hello'.trim()"), "\"Hello\"");
-    assert_eq!(forward(&mut context, "' \nHello'.trim()"), "\"Hello\"");
-    assert_eq!(forward(&mut context, "'Hello \n\r'.trim()"), "\"Hello\"");
-    assert_eq!(forward(&mut context, "' Hello '.trim()"), "\"Hello\"");
+    assert_eq!(forward(&mut context, r#"'Hello'.trim()"#), "\"Hello\"");
+    assert_eq!(forward(&mut context, r#"' \nHello'.trim()"#), "\"Hello\"");
+    assert_eq!(forward(&mut context, r#"'Hello \n\r'.trim()"#), "\"Hello\"");
+    assert_eq!(forward(&mut context, r#"' Hello '.trim()"#), "\"Hello\"");
```

Comment:
string `"'Hello \n\r'.trim()"` in Rust is evaluated as 
```
"'Hello
'.trim()"
``` 
in memory

and string literal `r#"'Hello \n\r'.trim()"#` in Rust is evaluated as `"'Hello \\n\\r'.trim()"` in memory

Reply:
You could escape the `\`, by using `\\n` in the source code, right?
But it can stay as is, it's just that you used for all 4 lines but it only affects 537 and 538, no?

Reply:
Yes, so do you suggest keeping line 536 and 539 as normal string literal?